portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,18 @@
+
 #content {
   margin-left: auto;
```

Comment:
Nit: These should be alphabetized.

Reply:
go/htmlcssstyle#Declaration_Order

Reply:
Unfortunately I do not have permission to access that document. Is this an accurate reference? https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Alphabetized the declarations in a commit. Quick question, what does "Nit" mean?